test/e2e/openliberty_test.go

```diff
@@ -30,16 +34,50 @@ func TestOpenLibertyApplication(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to add Trace scheme to framework: %v", err)
 	}
-
+	// basic tests that are runnable locally in minishift/kube
 	t.Run("OpenLibertyPullPolicyTest", OpenLibertyPullPolicyTest)
 	t.Run("OpenLibertyBasicTest", OpenLibertyBasicTest)
-	t.Run("OpenLibertyStorageTest", OpenLibertyBasicStorageTest)
-	t.Run("OpenLibertyPersistenceTest", OpenLibertyPersistenceTest)
 	t.Run("OpenLibertyProbeTest", OpenLibertyProbeTest)
 	t.Run("OpenLibertyAutoScalingTest", OpenLibertyAutoScalingTest)
+	t.Run("OpenLibertyStorageTest", OpenLibertyBasicStorageTest)
+	t.Run("OpenLibertyPersistenceTest", OpenLibertyPersistenceTest)
+	t.Run("OpenLibertyTraceTest", OpenLibertyTraceTest)
+
+
+	if cluster != "local" {
+		// only test non-OCP features on minikube
+		if cluster == "minikube" {
+			testIndependantFeatures(t)
+			return
+		}
+
+		// test all features that require some configuration
+		testAdvancedFeatures(t)
+		// test featurest hat require OCP
+		if cluster == "ocp" {
+			testOCPFeatures(t)
+		}
+	}
+
+
+}
+
+func testAdvancedFeatures(t *testing.T) {
+	// These features require a bit of configuration
+	// which makes them less ideal for quick minikube tests
 	t.Run("OpenLibertyServiceMonitorTest", OpenLibertyServiceMonitorTest)
 	t.Run("OpenLibertyKnativeTest", OpenLibertyKnativeTest)
-	t.Run("OpenLibertyTraceTest", OpenLibertyTraceTest)
-	t.Run("OpenLibertyDumpsTest", OpenLibertyDumpsTest)
 	t.Run("OpenLibertyServiceBindingTest", OpenLibertyServiceBindingTest)
+	t.Run("OpenLibertyCertManagerTest", OpenLibertyCertManagerTest)
+}
+
+// Verify functionality that is tied to OCP
+func testOCPFeatures(t *testing.T) {
+	t.Run("OpenLibertyImageStreamTest", OpenLibertyImageStreamTest)
+	t.Run("OpenLibertyDumpsTest", OpenLibertyDumpsTest)
```

Comment:
OL Dumps aren't open shift specific, so you can put `OpenLibertyDumpsTest` in the advanced features function. At the moment there isn't a functional difference, but going forward there could be depending on what environments we test and how we configure minikube in travis.